tangelo/linq/tests/test_simulator.py

```diff
@@ -147,6 +147,43 @@ def test_get_exp_value_mixed_state(self):
             results[b] = sim.get_expectation_value(op1, circuit_mixed)
             np.testing.assert_almost_equal(results[b], reference, decimal=2)
 
+    def test_get_variance(self):
+        """ Test variance for simple analytical circuit. """
+
+        opx = 1.0 * QubitOperator("X0")
+        opy = 1.0 * QubitOperator("Y0")
+        opz = 1.0 * QubitOperator("Z0")
+
+        # prepares sqrt(2/3)|0> + -i*sqrt(1/3)|1>
+        circuit = Circuit([Gate("RX", 0, parameter=2*np.arcsin(np.sqrt(1/3)))])
+
+        for b in installed_simulator:
```

Comment:
@AlexandreF-1qbit @JamesB-1qbit Since all the variance functions  have been implemented in `SimulatorBase` and rely on the existing and thoroughly tested `get_expectation_value` method each backend implements: is there any meaning in testing this for all simulators ?

Couldn't we just use the default cirq simulator for that test ? We can then maybe crank it up to 10**6 shots to guarantee the comparison up to 2 decimals never fails because of "bad luck" (happens on rare occasions in other tests for me in my recent developments).

Reply:
We use just the cirq simulator now.